csuntechlab/iSTART
# STR-203 -- Update Quiz Incorrect and Correct Indicators

# Description
Updated the indicator background colors and checked for consistency with the mock-up.
  
# Testing
1. Visit any `quiz slide` in any module.
2. Ensure when selecting a wrong answer, the indicator background is red.
3. Ensure when selecting a correct answer, the indicator background is green.
4. Make sure all the quiz functionality works.
5. Compare the mock-up with the quiz content.

# Note
1. Indicator slide animation.
2. Change border to indicator color & transition.
